pkg/scheduler/framework/plugins/noderesources/resource_allocation.go

```diff
@@ -100,7 +99,7 @@ func calculateResourceAllocatableRequest(nodeInfo *framework.NodeInfo, pod *v1.P
 		}
 	}
 	if klog.V(10).Enabled() {
-		klog.Infof("requested resource %v not considered for node score calculation", resource)
+		klog.InfoS("Requested resource is omitted for node score calculation", "resource", resource)
```

Comment:
The resource here points to the ResourceName. Can we use "resourceName" as the key.

Reply:
Done, thanks pchan!!